at_client/src/test/java/org/atsign/common/ByteUtilTest.java

```diff
@@ -0,0 +1,90 @@
+package org.atsign.common;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+public class ByteUtilTest {
+
+
+    @Test
+    public void testByteUtil0(){
+        ByteUtil byteUtil = new ByteUtil();
```

Comment:
Once you've made them into static methods, you can get rid of this line because you don't need to create an object to use the class.